<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioPlayer</class>
 <widget class="QWidget" name="AudioPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player de Áudio</string>
  </property>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/button_blue_stop.png</normaloff>:/i/button_blue_stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_play">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/button_blue_play.png</normaloff>:/i/button_blue_play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="Phonon::VolumeSlider" name="volume_slider">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>114</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="Phonon::SeekSlider" name="seek_slider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_audio_description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 12pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Nenhum áudio selecionado...</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
